src/privacy_accounting/analysis/privacy_ledger.py

```diff
@@ -75,7 +75,7 @@ def record_sum_query(self, l2_clipping_bound, noise_stddev):
         self._query_count = self._query_count + 1
         self._query_buffer.append(torch.Tensor([self._sample_count, l2_clipping_bound, noise_stddev]))
 
-    def finalise_sample(self):
```

Comment:
This should be sample - a minibatch is a single sample you draw from the data set to apply a query to.